lightkurve/tests/test_search.py

```diff
@@ -295,3 +295,14 @@ def test_indexerror_631():
     # This previously triggered an exception:
     result = search_lightcurvefile("KIC 8462852", sector=15)
     assert len(result) == 1
+    len(search) == 0
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expression "len(search) == 0" is assigned to nothing](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=3940605)